source/uwp/Renderer/lib/BaseRegistration.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
```

Comment:
Looks like this file isn't added to the solution

Reply:
Looks like this didn't get used, so we should remove it from the PR

---
In reply to: [141433885](https://github.com/Microsoft/AdaptiveCards/pull/695#discussion_r141433885) [](ancestors = 141433885)

Reply:
It's not supposed to even be in the CR. Removing it :P This was one of my bad attempts to getting a templated registration to work.

---
In reply to: [141433885](https://github.com/Microsoft/AdaptiveCards/pull/695#discussion_r141433885) [](ancestors = 141433885)